src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -164,8 +173,8 @@ public void configMotors() {
 		// current limit
 		launcherTopMotor.setSmartCurrentLimit(40);
 		launcherBottomMotor.setSmartCurrentLimit(40);
-		launcherAngleOneMotor.setSmartCurrentLimit(80);
-		launcherAngleTwoMotor.setSmartCurrentLimit(80);
+		launcherAngleOneMotor.setSmartCurrentLimit(60);
+		launcherAngleTwoMotor.setSmartCurrentLimit(60);
```

Comment:
Have we tested these updated current limits on the robot?

Reply:
yes